[doc][ai-inference-rest] broken link

Summary of broken links:
'file:///D:/a/_work/1/s/sdk/ai/ai-inference-rest/README.md' has 1 broken link(s):
  https://aka.ms/AAp1kxa
Checked 32219 links with 1 broken link(s) found.
##[error]PowerShell exited with code '1'.


![Image](https://github.com/user-attachments/assets/0ea9f258-f55d-41a2-92f1-97eb43a9ab77)

--- a/sdk/ai/ai-inference-rest/README.md
+++ b/sdk/ai/ai-inference-rest/README.md
@@ -8,7 +8,7 @@ Key links:
 
 - [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/ai/ai-inference-rest)
 - [Package (NPM)](https://aka.ms/npm-azure-rest-ai-inference)
-- [API reference documentation](https://aka.ms/AAp1kxa)
+- [API reference documentation](https://learn.microsoft.com/javascript/api/@azure-rest/ai-inference/)
 - [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/ai/ai-inference-rest/samples)
 
 ## Getting started

